src/DependencyInjection/Configuration.php

```diff
@@ -226,7 +226,7 @@ public function getConfigTreeBuilder()
                                 ->children()
                                     ->scalarNode('label')->end()
                                     ->scalarNode('label_catalogue')->end()
-                                    ->scalarNode('icon')->defaultValue('<i class="fa fa-folder"></i>')->end()
+                                    ->scalarNode('icon')->end()
```

Comment:
Why change this?

Reply:
If we automatically set a default value to `icon`, it can't fallback on `default_icon` value.

`<i class="fa fa-folder"></i>` is already the default value of `default_icon`. The icon will fallback to `default_icon` which will fallback on `<i class="fa fa-folder"></i>`.